config.py

```diff
@@ -1985,11 +1985,20 @@ def _get_reject(
         # Automated threshold calculation requested
         import autoreject
 
+        ch_types_autoreject = list(ch_types)
+        if 'meg' in ch_types_autoreject:
+            ch_types_autoreject.remove('meg')
+            if 'mag' in epochs:
+                ch_types_autoreject.append('mag')
+            if 'grad' in epochs:
+                ch_types_autoreject.append('grad')
```

Comment:
@apmellot this should fix your problem on camcan

Reply:
Could you elaborate why this is needed? Doesn't `meg` do the right thing?

Reply:
autoreject does not support "meg"